common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockHandler.java

```diff
@@ -143,4 +179,26 @@ private void checkAuth(TransportClient client, String appId) {
     }
   }
 
+  /**
+   * A simple class to wrap all shuffle service wrapper metrics
+   */
+  private class ShuffleMetrics implements MetricSet {
+    private final Map<String, Metric> allMetrics;
+    private final Timer timeDelayForOpenBlockRequest = new Timer();
+    private final Timer timeDelayForRegisterExecutorRequest = new Timer();
+    private final Meter transferBlockRate = new Meter();
```

Comment:
Can you add comments describing the metrics and their units? e.g. bytes, milliseconds

Also consider renaming them for clarity, I think `openBlockRequestLatencyMillis`, `registerExecutorRequestLatencyMillis`, `blockTransferRateBytes` would be more clear to the reader.